terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf

```diff
@@ -216,8 +215,7 @@ module "proxy_containerized" {
     password = "admin"
   }
   runtime = "podman"
-  # Temporary using the correct URL for the 5.0.1 BV only. See https://github.com/SUSE/spacewalk/issues/25120
-  container_repository = "registry.suse.de/suse/maintenance/35237/suse_sle-15-sp6_update_products_manager50_update_containerfile/suse/manager/5.0/x86_64"
+  container_repository = "registry.suse.de/suse/sle-15-sp6/update/products/manager50/containerfile"
```

Comment:
I think for the proxy, we should leave the URL as is. See e.g. Kostas who also struggled with it: https://github.com/SUSE/susemanager-ci/pull/1325/files. Same for the other proxy changes.

Reply:
So maybe my hypothesis about mgradm and mgrpxy being different was right :trollface: 

ok

Reply:
According to Kostas:

>  This url works, the mgradm appends the namespace end, but mgrpxy doesn't.

So you were right. mgradm and mgrpxy are not alinged.